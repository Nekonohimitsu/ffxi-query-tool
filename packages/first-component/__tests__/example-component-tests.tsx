import renderer from 'react-test-renderer';
import ExampleComponent from '../src/example-component';
import React from 'react';

describe('Basic Tests', () => {
    test('it should handle my example component', () => {
        const component = renderer.create(<ExampleComponent id={2} />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
