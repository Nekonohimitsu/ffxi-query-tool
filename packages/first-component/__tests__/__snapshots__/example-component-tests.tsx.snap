// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic Tests it should handle my example component 1`] = `
<div>
  2
</div>
`;
